miso-web/src/main/webapp/pages/editSample.jsp

```diff
@@ -739,16 +749,7 @@
 	      <button id="makeTable" class="disabled" disabled="disabled" onclick="Sample.hot.makeNewSamplesTable();">Make Table</button>
 	    </div>
 	  </div>
-    <div class="clear">
-      <c:choose>
-      <c:when test="${detailedSample}">
-        <button id="saveDetailed" class="disabled" disabled="disabled" onclick="Sample.hot.saveDetailedData();">Save</button>
-      </c:when>
-      <c:otherwise>
-        <button id="savePlain" onclick="Sample.hot.savePlainData();">Save Bulk</button>
-      </c:otherwise>
-      </c:choose>
-    </div>
+    <div class="clear"></div>
```

Comment:
Here's the rendering without it, just for fun: ![cleardiv](https://cloud.githubusercontent.com/assets/8324996/17564174/8d2603d4-5f00-11e6-9d94-23e85faa37d6.png)